docs/UserGuide.md

```diff
@@ -399,9 +400,19 @@ If your changes to the data file makes its format invalid, UGTeach will discard
 Furthermore, certain edits can cause the application to behave in unexpected ways (e.g., if a value entered is outside the acceptable range). Therefore, edit the data file only if you are confident that you can update it correctly.
 </box>
 
+
 --------------------------------------------------------------------------------------------------------------------
+
 <div style="page-break-after: always;"></div>
 
+## Glossary
+
+* **Duplicated student:** Two students are duplicates if they have the exact same **NAME** and the same **PHONE NUMBER** at the same time.
```

Comment:
Even though its mentioned in the constraints for name in the `add` command, I think we can mention that the name is case insensitive to remind the reader here